django/contrib/admin/tests.py

```diff
@@ -26,17 +57,27 @@ class AdminSeleniumWebDriverTestCase(StaticLiveServerTestCase):
         'django.contrib.sessions',
         'django.contrib.sites',
     ]
-    webdriver_class = 'selenium.webdriver.firefox.webdriver.WebDriver'
+
+    @classmethod
+    def _get_local_webdriver_class(self, spec):
+        """
+        Return the webdriver for a local browser corresponding to the given
+        browser spec.
+        """
+        driver = 'selenium.webdriver.%s.webdriver.WebDriver' % spec
```

Comment:
This will work only for selenium version greater than 2.45, most recent version is 2.51. Updated docs.